src/migration/1641894827503-Initial.ts

```diff
@@ -0,0 +1,200 @@
+        await queryRunner.query(`ALTER TABLE "payload_decoder" ADD CONSTRAINT "FK_ce6313b2fa3de41f2d4d9409003" FOREIGN KEY ("updatedById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "payload_decoder" ADD CONSTRAINT "FK_20c47c4f5719e0d398743401acd" FOREIGN KEY ("organizationId") REFERENCES "organization"("id") ON DELETE CASCADE ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "sigfox_group" ADD CONSTRAINT "FK_f5401745b31e0eedff2d219f77d" FOREIGN KEY ("createdById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "sigfox_group" ADD CONSTRAINT "FK_23d9047d051ab1ad8875306ae98" FOREIGN KEY ("updatedById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "sigfox_group" ADD CONSTRAINT "FK_a38129f018eb963aea886de12e9" FOREIGN KEY ("belongsToId") REFERENCES "organization"("id") ON DELETE CASCADE ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "organization" ADD CONSTRAINT "FK_acdbd1e490930af04b4ff569ca9" FOREIGN KEY ("createdById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "organization" ADD CONSTRAINT "FK_a7a6b96e8460aeb314599cf3580" FOREIGN KEY ("updatedById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "device_model" ADD CONSTRAINT "FK_9d26f066b617bbdac41d72e82b2" FOREIGN KEY ("createdById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "device_model" ADD CONSTRAINT "FK_ed6727d1c4a7f9e6299ba6e2961" FOREIGN KEY ("updatedById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "device_model" ADD CONSTRAINT "FK_75a9d1c5495facf583614ca2b5f" FOREIGN KEY ("belongsToId") REFERENCES "organization"("id") ON DELETE CASCADE ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "lorawan-multicast" ADD CONSTRAINT "FK_bfb223faca94d9e217c39439c49" FOREIGN KEY ("createdById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "lorawan-multicast" ADD CONSTRAINT "FK_c776cdd3a4b1818c0995ee60baa" FOREIGN KEY ("updatedById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "multicast" ADD CONSTRAINT "FK_60c29583cfbafbd6b0e1a9679c9" FOREIGN KEY ("createdById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
+        await queryRunner.query(`ALTER TABLE "multicast" ADD CONSTRAINT "FK_0000d423c968ec8ba68482f8b0b" FOREIGN KEY ("updatedById") REFERENCES "user"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
```

Comment:
Everything with multicast should be included in a seperate migration and not in the init migration